helm/robusta/templates/_helpers.tpl

```diff
@@ -54,19 +54,49 @@ active_playbooks:
   {{- fail "The `playbooks` value is deprecated. Rename `playbooks`  to `customPlaybooks` and remove builtin playbooks which are now defined separately" -}}
 {{- end }}
 
-{{- if .Values.priorityBuiltinPlaybooks }}
-{{ toYaml .Values.priorityBuiltinPlaybooks | indent 2 }}
+{{- $disabledPlaybooks := .Values.disabledPlaybooks }}
+
+{{- $myplaybooks := .Values.priorityBuiltinPlaybooks }}
```

Comment:
Can you please rename to `priorityPlaybooks`
Just naming, but might make this easier to read